### PR TITLE
Multiple studyify cancer type summary and mutex

### PR DESCRIPTION
**Pull request by adamabeshouse**

### CODE REVIEW

File: src/pages/resultsView/ResultsViewPageStore.ts [hunk 1]
```diff
@@ -193,16 +193,16 @@ export class ResultsViewPageStore {
...
 
-    readonly molecularData = remoteData({
+    readonly molecularData = remoteData<GeneMolecularData[]>({
         await: () => [
             this.studyToDataQueryFilter,
             this.genes,
             this.selectedMolecularProfiles
         ],
-        invoke: async() => {
+        invoke: () => {
```
Comment (alisman) [inline src/pages/resultsView/ResultsViewPageStore.ts]: why take away async?

Comment (adamabeshouse) [reply src/pages/resultsView/ResultsViewPageStore.ts]: for code legibility, since nothing async is actually happening

Comment (alisman) [reply src/pages/resultsView/ResultsViewPageStore.ts]: we do call api here though.  215

Comment (adamabeshouse) [reply src/pages/resultsView/ResultsViewPageStore.ts]: but we're returning the promise, theres no awaiting going on inside

**Pull request merged**